app/auth/UnauthenticatedController.scala

```diff
@@ -0,0 +1,19 @@
+package auth
+
+import org.pac4j.core.profile.{CommonProfile, ProfileManager}
+import org.pac4j.play.PlayWebContext
+import org.pac4j.play.scala.{Security, SecurityComponents}
+import play.api.mvc.{AnyContent, Request}
+
+import javax.inject.Inject
+
+class UnauthenticatedController @Inject ()(val controllerComponents: SecurityComponents) extends Security[CommonProfile] {
```

Comment:
I'm wondering about the name for this, because I started interpreting it as a controller for 401 errors or something. But it looks like it's for pages which don't need to authenticate the user - is that right? It sounds odd to call it "unauthenticated" when the user might actually be authenticated - it just doesn't matter whether they are or not.

I'm trying to think of alternatives. `PublicPageController`? `UnprotectedPageController`? The "page" bit isn't very nice, but it sounds even stranger without it. I wonder if anyone has any better ideas on Slack?

Reply:
I see your point. `UnprotectedPageController` sounds right, I've changed it to that.